### PR TITLE
Mobile menu javascript

### PR DESCRIPTION
**Pull request by 0zedd**
This feature javascript was added to mobile version menu

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -20,6 +18,49 @@
...
+}
+
+.nav_expand {
+  position: absolute;
+  height: 100%;
+  width: 375px;
+  background-color: #6070ff;
+  opacity: 0.92;
+  display: none;
+}
```
Comment (leonardodiasb) [inline style.css]: - [x] On some screen sizes the mobile menu is not covering the entire screen. Kindly fix that.
image

File: index.html [hunk 2]
```diff
@@ -14,10 +14,18 @@
...
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+        </ul>
+        <div class="nav_expand" id="hide">
+            <ul class="expand_ims">   
+            <li><img class="iconx" src="./img/x.svg" alt="icon"></li>
+            <li id="ims">About</li>
             <li>Contact</li>
+            <li>Portfolio</li>
```
Comment (leonardodiasb) [inline index.html]: - [x] Kindly use the font family provided by the Figma template for the mobile menu buttons.
- [x] When the user clicks on the menu buttons they should display the correct part of the page and close the mobile menu. Kindly fix that.
image

File: index.html [hunk 3]
```diff
@@ -14,10 +14,18 @@
...
         <ul class="nav_ul">
-            <li>Portfolio</li>
-            <li>About</li>
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+        </ul>
+        <div class="nav_expand" id="hide">
+            <ul class="expand_ims">   
+            <li><img class="iconx" src="./img/x.svg" alt="icon"></li>
```
Comment (leonardodiasb) [inline index.html:23]: - [x] Kindly set the margin to match the margins and padding provided by the Figma template for your X button and your hamburger.
Your design:
image

Figma design:
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @0zedd , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README is descriptive ✔️

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -18,6 +18,14 @@
             <li>About</li>
             <li>Contact</li>
         </ul>
+        <div class="nav_expand" id="hide">
+            <ul class="expand_ims">
+            <li><img class="iconx" src="./img/x.svg" alt="icon"></li>
+            <li id="ims"><a href="#about">About</a></li>
+            <li><a href="#contact">Contact</a></li>
+            <li><a href="#port">Portfolio</a></li>
```
Comment (ShahierNashaat) [inline index.html]: - [x] Please implement the changes that were required by the previous code reviewer as when the click on one of the menu items the mobile menu should disappear.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @0zedd , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 

## Hi team👋, 

Well done implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Highlights 🎯 
✔️correct GitHub Flow 
✔️Good commit history
✔️Linter checks

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**